commit 04483260f07b2eef5045646ff96a90afbf214c54
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-08-12T16:21:41+00:00 (8 weeks ago)
    
    OS-7948 /usr/lib/fs/smbfs/mount shipped with wrong permissions

diff --git a/manifest b/manifest
index 6c8fc24548..27e93b9038 100644
--- a/manifest
+++ b/manifest
@@ -5897,9 +5897,9 @@ s usr/lib/fs/smbfs/amd64/libshare_smbfs.so=libshare_smbfs.so.1
 f usr/lib/fs/smbfs/dfshares 0555 root bin
 f usr/lib/fs/smbfs/libshare_smbfs.so.1 0755 root sys
 s usr/lib/fs/smbfs/libshare_smbfs.so=libshare_smbfs.so.1
-f usr/lib/fs/smbfs/mount 0555 root bin
+f usr/lib/fs/smbfs/mount 4555 root bin
 f usr/lib/fs/smbfs/share 0555 root bin
-f usr/lib/fs/smbfs/umount 0555 root bin
+f usr/lib/fs/smbfs/umount 4555 root bin
 f usr/lib/fs/smbfs/unshare 0555 root bin
 d usr/lib/fs/tmpfs 0755 root sys
 f usr/lib/fs/tmpfs/mount 0555 root bin
